GSoC/2016/README.md

```diff
@@ -0,0 +1,33 @@
+# Notes for 2016 GSoC application
+
+It's my first time applying for the GSoC program. For lack of experiences, I missed the deadline of submitting my enrollment letter, which finally caused the disapproval.
+
+I didn't know the requirement of enrollment letter until I checked the GSoC website one day before the deadline. Unfortunately, this proposal was not approved only because of the miss of deadline in submitting a supporting letter. But the proposal itself has passed organizational peer reviews and could be useful for future reference.
+
+In any case, the project has to move on. For your interest, please check [QuCmp.jl](https://github.com/JuliaQuantum/QuCmp) in JuliaQuantum's repo.
+
+To prevent the same tragedy happening on future applicants, especially to help applicants in regions without Google services, I am leaving some advice below.
+
+### For students
+
+- If you have trouble in connecting GSoC's official website, eg. the browser gives you 404 error, you should ask JuliaQuantum members for help by email or gitter. Make sure you have read the rules and regulations before applying, with all the deadlines in mind.
+- Check your application status regularly after your application.
+- if you do not use gmail very often or you have trouble in using gmail, please set your gmail automatically forward the emails to your commonly used email
+- If you do not use gmail very often or you can hardly use gmail, please set your gmail to automatically forward emails to your commonly used email address. The organizers from Google will send emails to your Gmail address by default.
+- Prepare your enrollment letter early before the deadline, which could take some time and has to be your university's official certificate in English.
+
+### For mentors
+
+By courtesy of JuliaQuantum's policy that non-organizational person can join the organization by passing a qualification review process and act as a mentor of the applicant, it is possible for outsiders to become a GSoC mentor under the organization's umbrella. Again, if there is any communication problems with Google or understanding the rules, the organization could offer a hand.
+
+- You may have trouble in reading through the mentor's guide and rules in GSoC's official website. If so, please feel free to ask in JuliaQuantum's email list or gitter channel for help.
```

Comment:
Don't forget to delete this paragraph as I have commented earlier. I think it's repeating and over-shooting. Thanks.